src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -1151,6 +1151,7 @@ export function generateNumericalData(numericalBins: DataBin[]): BarDatum[] {
             // no need to add 1 (no interval needed for the previous value)
             if (
                 index - 1 > -1 &&
+                numericalBins[index - 1].start &&
```

Comment:
this is going to return false if start = 0 - need to explicitly check start !== undefined or null if that's what's desired